ui/app/src/components/RejectDialog/RejectDialogContainer.tsx

```diff
@@ -32,7 +32,7 @@ import { batchActions } from '../../state/actions/misc';
 export function RejectDialogContainer(props: RejectDialogContainerProps) {
   const { items, onClose, onRejectSuccess, isSubmitting } = props;
   const [checkedItems, setCheckedItems] = useState([]);
-  const [rejectionReason, setRejectionReason] = useState('');
+  const [rejectionReason, setRejectionReason] = useState('custom');
```

Comment:
I don't see changes on the `onRejectionReasonChanges` to avoid fetching `custom`.
- Let's change `custom` for something more specific to avoid config file key collision (e.g. `typeCustomReason`)
- On the comment fetcher, make sure the `value !== 'typeCustomReason'` before fetching
- Rename `onRejectionReasonChanges` => `onRejectionReasonChange`
- Rename `RejectDialogUIProps['setRejectionReason']` same as above
- Rename `RejectDialogUIProps['setRejectionComment']` to `onCommentChange`